package.json

```
@@ -47,7 +47,7 @@
     "index.js"
   ],
   "dependencies": {
-    "@types/unist": "^2.0.0",
+    "@types/unist": "^2.0.5",
```

@ChristianMurphy Should I keep this at `2.0.0` as you suggest on unified?

---

yes, the minimum version in range please

```suggestion
    "@types/unist": "^2.0.0",
```